ansible_rulebook/rule_set_runner.py

```
@@ -189,10 +189,14 @@ async def _cleanup(self):
                 )
             )
-        stats = lang.end_session(self.name)
-        if self.parsed_args and self.parsed_args.heartbeat > 0:
-            await send_session_stats(self.event_log, stats)
-        logger.info(pformat(stats))
+        try:
+            stats = lang.end_session(self.name)
+            if self.parsed_args and self.parsed_args.heartbeat > 0:
+                await send_session_stats(self.event_log, stats)
+            logger.info(pformat(stats))
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)